pkg/radius/handler/3gpp_types.go

```diff
@@ -252,10 +257,10 @@ func UnmarshalEAP5GData(codedData []byte) (eap5GMessageID uint8, anParameters *A
 
 							if len(parameterValue) < int(parameterLength) {
 								return 0, nil, nil, errors.New("Error formatting")
-							} else {
-								parameterValue = parameterValue[:parameterLength]
 							}
-
+							// else {
+							// 	parameterValue = parameterValue[:parameterLength]
+							// }
```

Comment:
comment by mistake?

Reply:
parameterValue is unused variable in this case. Should I delete these code?

Reply:
ok, deleted.

Reply:
I have asked the senior and he said that I can ignore these codes.